packages/mongo-nav/src/types.ts

```diff
@@ -1,18 +1,50 @@
+const ActiveNavElement = {
+  OrgNavOrgSettings: 'orgNavOrgSettings',
```

Comment:
We should at least do 
```suggestion
  OrgNavOrgSettings: NavElement.OrgNavOrgSettings,
```
etc

Reply:
👍